cgmes/src/main/resources/config.yml

```diff
@@ -7,8 +7,13 @@ componentDefaultConfig:
 #    localDir: <path_to_local_tmp>
 #    availableCore: 1
 #
-#hades2:
-#  homeDir: <path_to_hades_2>
-#  timeout: 120
-#  debug: true
-#  useSAX: false
\ No newline at end of file
+hades2:
+  homeDir: /home/tilloyann/simulators/hades2/hades2-V6.4.0.1.1/
+  timeout: 120
+  debug: true
+  useSAX: false
+
+hades2-default-parameters:
+  balanceType: PROPORTIONAL_TO_LOAD
+  computedConnectedComponentNumber: 1
+  remoteVoltage: true
```

Comment:
It seems to me the configuration should remain commented, shouldn't it?

Reply:
I think this modification should be reverted because, this PR is about VL topology, not about CGMES. Is it a mistake?

Reply:
It is a mistake.